src/DoctrineEventStore.php

```diff
@@ -254,7 +284,7 @@ private function commitEvent(StreamName $streamName, Event $event, Version $vers
                 'recordedat' => $this->clock->now(),
             ],
             [
-                'version' => Types::INTEGER,
+                'version' => Types::BIGINT,
```

Comment:
This is technically wrong in sqlite I guess? But it shouldn't matter much, so no need to change IMHO given that it works.

Reply:
eagle-eye Christian :)

The type comes in relevant in [Connection:getBindingInfo()](https://github.com/doctrine/dbal/blob/6a793fb72948c9dec844de57de8cdbc16ff75bec/src/Connection.php#L1849-L1850)

For the value itself, it does not make a difference, because neither `IntegerType` nor `BigIntType` implement an overload for [convertToDatabaseValue()](https://github.com/doctrine/dbal/blob/6a793fb72948c9dec844de57de8cdbc16ff75bec/src/Types/Type.php#L69-L72)

But I realized, that the returned `$bindingType` is different (`INTEGER` for [IntegerType](https://github.com/doctrine/dbal/blob/6a793fb72948c9dec844de57de8cdbc16ff75bec/src/Types/IntegerType.php#L46-L49) and `STRING` for [BigIntType](https://github.com/doctrine/dbal/blob/6a793fb72948c9dec844de57de8cdbc16ff75bec/src/Types/BigIntType.php#L32-L35)).

I'm not 100% sure whether it makes a difference in our case...